src/Features/Core/Portable/CodeFixes/Suppression/WrapperCodeFixProvider.cs

```diff
@@ -12,10 +12,10 @@ internal sealed class WrapperCodeFixProvider : CodeFixProvider
         private readonly ImmutableArray<string> _originalDiagnosticIds;
         private readonly ISuppressionFixProvider _suppressionFixProvider;
 
-        public WrapperCodeFixProvider(ISuppressionFixProvider suppressionFixProvider, ImmutableArray<Diagnostic> originalDiagnostics)
+        public WrapperCodeFixProvider(ISuppressionFixProvider suppressionFixProvider, ImmutableArray<string> diagnosticIds)
```

Comment:
okay makes sense, probably you can leave the .Distinct() call inside the ctor as the caller really shouldn't need to worry about it?